### DISCUSSION TITLE
Kimai 1.16.8 fails to install with ClassNotFound Error on Ubunut 20.4

### DISCUSSION THREAD

**Discussion by mandelain** [kimai/kimai#3017] [On-Premise: Installation & Updates] [answered]
I have gotten to the point of installation ([here](https://www.kimai.org/documentation/fresh-ubuntu-20.html)), post cloning the git repo for 1.16.8, when I run:
`bin/console kimai:install -n `

I recieve the following error:
_Symfony\Component\ErrorHandler\Error\ClassNotFoundError^ {#41
  #message: """
    Attempted to load class "DoctrineFixturesBundle" from namespace "Doctrine\Bundle\FixturesBundle".\n
    Did you forget a "use" statement for another namespace?
    """
  #code: 0
  #file: "/var/www/kimai2/src/Kernel.php"
  #line: 106
  trace: {
    /var/www/kimai2/src/Kernel.php:106 {
      App\Kernel->registerBundles()^
      › if (isset($envs['all']) || isset($envs[$this->environment])) {
      ›     yield new $class();
      › }
    }
    /var/www/kimai2/vendor/symfony/http-kernel/Kernel.php:446 { …}
    /var/www/kimai2/vendor/symfony/http-kernel/Kernel.php:133 { …}
    /var/www/kimai2/vendor/symfony/framework-bundle/Console/Application.php:169 { …}
    /var/www/kimai2/vendor/symfony/framework-bundle/Console/Application.php:75 { …}
    /var/www/kimai2/vendor/symfony/console/Application.php:149 { …}
    /var/www/kimai2/bin/console:29 { …}
  }
}
2021-12-12T20:48:59+11:00 [critical] Uncaught Error: Class 'Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle' not found_

Any thoughts would be appreciated.

**Comment by kevinpapst** [comment]
Seems you installed composer packages for prod environment and try to run Kimai in APP_ENV=dev ... check your .env file

**Reply by mandelain** [reply]
I have checked the .env file, it only contains the DATABASE_URL=[url] line

**Comment by kevinpapst** [comment, marked as answer]
Then you should fix that. The .env file needs the entries which were there initially:
https://github.com/kevinpapst/kimai2/blob/master/.env.dist

**Reply by mandelain** [reply]
this was my problem, thanks a lot, I appreciate the help!
